### DISCUSSION TITLE
Deploy Problem

### DISCUSSION THREAD

**Discussion by yahia1997ezat** [pmndrs/react-spring#1237] [Support]
Hi,
I'm using react-three-fiber, in addition, @react-spring/three, react-@spring/web, and react-spring
things working great locally but when I build/deploy my app I encounter this problem

`Uncaught TypeError: r.willAdvance is not a function
    at FrameLoop.ts:173
    at Module.gu (react-dom.production.min.js:244)
    at a.m.advance (FrameLoop.ts:169)
    at Array.<anonymous> (index.ts:12)
    at Me (web.js:96)`

I used create-react-app CLI to create my app
can you help me please it's a little bit urgent

**Comment by tomdohnal** [comment]
See https://github.com/pmndrs/react-spring/issues/1078#issuecomment-677528907 or other comments in the discussion for workaround
